src/llmq/instantsend.cpp

```diff
@@ -854,7 +854,7 @@ bool CInstantSendManager::ProcessPendingInstantSendLocks()
     const CBlockIndex* pBlockIndexTip = WITH_LOCK(cs_main, return ::ChainActive().Tip());
     if (pBlockIndexTip && utils::GetInstantSendLLMQType(qman, pBlockIndexTip) == Params().GetConsensus().llmqTypeDIP0024InstantSend) {
         // Don't short circuit. Try to process deterministic and not deterministic islocks
-        return ProcessPendingInstantSendLocks(true) & ProcessPendingInstantSendLocks(false);
+        return ProcessPendingInstantSendLocks(true) && ProcessPendingInstantSendLocks(false);
```

Comment:
the comment specifically talks about not wanting to short circuit. but we can refactor this hard by hardening dip24 and remove non-deterministic islocks